data/obs-packaging/cc-oci-runtime/README.md

```diff
@@ -8,10 +8,14 @@ With these files we generated Fedora and Ubuntu packages for this component.
 
 ``./update_runtime.sh [VERSION]``
 
+``VERSION`` parameter is optional, it can be a tag, a branch or a git hash
+
+If no ``VERSION`` is specified to the script, it will get it from the one 
+specified in the ``configure.ac`` file found at the the repository's root.
```

Comment:
I'd simplify and just say something like:

If ``VERSION`` is not specified, it will be determined automatically from the top-level ``configure.ac`` file.